node/src/components/consensus/protocols/highway/round_success_meter.rs

```diff
@@ -178,3 +180,106 @@ impl<C: Context> RoundSuccessMeter<C> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::components::consensus::{
+        cl_context::ClContext,
+        protocols::highway::round_success_meter::{
+            ACCELERATION_PARAMETER, MAX_FAILED_ROUNDS, NUM_ROUNDS_TO_CONSIDER,
+        },
+    };
+
+    const TEST_ROUND_EXP: u8 = 13;
+    const TEST_MIN_ROUND_EXP: u8 = 8;
+    const TEST_MAX_ROUND_EXP: u8 = 19;
+
+    #[test]
+    fn new_exponent_steady() {
+        let round_success_meter: super::RoundSuccessMeter<ClContext> =
+            super::RoundSuccessMeter::new(
+                TEST_ROUND_EXP,
+                TEST_MIN_ROUND_EXP,
+                TEST_MAX_ROUND_EXP,
+                crate::types::Timestamp::now(),
+            );
+        assert_eq!(round_success_meter.new_exponent(), TEST_ROUND_EXP);
+    }
+
+    #[test]
+    fn new_exponent_slow_down() {
+        let mut round_success_meter: super::RoundSuccessMeter<ClContext> =
+            super::RoundSuccessMeter::new(
+                TEST_ROUND_EXP,
+                TEST_MIN_ROUND_EXP,
+                TEST_MAX_ROUND_EXP,
+                crate::types::Timestamp::now(),
+            );
+        // If there have been more rounds of failure than MAX_FAILED_ROUNDS, slow down
+        round_success_meter.rounds = vec![false; MAX_FAILED_ROUNDS + 1].into();
+        assert_eq!(round_success_meter.new_exponent(), TEST_ROUND_EXP + 1);
+    }
+
+    #[test]
+    fn new_exponent_can_not_slow_down_because_max_round_exp() {
+        // If the round exponent is the same as the maximum round exponent, can't go up
+        let mut round_success_meter: super::RoundSuccessMeter<ClContext> =
+            super::RoundSuccessMeter::new(
+                TEST_MAX_ROUND_EXP,
+                TEST_MIN_ROUND_EXP,
+                TEST_MAX_ROUND_EXP,
+                crate::types::Timestamp::now(),
+            );
+        // If there have been more rounds of failure than MAX_FAILED_ROUNDS, slow down -- but can't
+        // slow down because of ceiling
+        round_success_meter.rounds = vec![false; MAX_FAILED_ROUNDS + 1].into();
+        assert_eq!(round_success_meter.new_exponent(), TEST_MAX_ROUND_EXP);
+    }
+
+    #[test]
+    fn new_exponent_speed_up() {
+        // If there's been enough successful rounds and its an acceleration round, speed up
```

Comment:
```suggestion
        // If there's been enough successful rounds and it's an acceleration round, speed up
```

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/479/commits/8e16624e1fb6eac81369e4ed6679272be7fe2ac1